arm/Microsoft.Network/expressRouteCircuits/readme.md

```diff
@@ -7,15 +7,15 @@ This template deploys a ExrepressRoute Circuit.
 | Resource Type | Api Version |
 | :-- | :-- |
 | `Microsoft.Authorization/locks` | 2016-09-01 |
+| `Microsoft.Authorization/roleAssignments` | 2020-04-01-preview |
 | `Microsoft.Insights/diagnosticSettings` | 2017-05-01-preview |
 | `Microsoft.Network/expressRouteCircuits` | 2021-02-01 |
-| `Microsoft.Network/expressRouteCircuits/providers/roleAssignments` | 2021-04-01-preview |
 
 ## Parameters
 
 | Parameter Name | Type | Default Value | Possible Values | Description |
 | :-- | :-- | :-- | :-- | :-- |
-| `bandwidthInMbps` | int |  |  | Required. This is the bandwidth in Mbps of the circuit being created. It must exactly match one of the available bandwidth offers List ExpressRoute Service Providers API call. SkuTier 'Local' has special bandwith requirements. It requires minimum 1000mbps.|
+| `bandwidthInMbps` | int |  |  | Required. This is the bandwidth in Mbps of the circuit being created. It must exactly match one of the available bandwidth offers List ExpressRoute Service Providers API call. |
```

Comment:
Outside of scope but still: Any chance you could add a `@minValue(1000)` to the `bandwidthInMbps` in the _deploy.bicep_ file?

Reply:
I can, but it says `SkuTier 'Local' has special bandwidth requirements. It requires minimum 1000mbps`. Are these two sentences linked? Is the limit conditional?

Reply:
Just checked the template reference. Doesn't seem like this sentence exists there. Beggs the question if that limitation actually exists

Reply:
Nope it does not. Please ignore